cmd/create/cluster/cmd.go

```diff
@@ -287,6 +288,13 @@ func init() {
 		"A proxy URL to use for creating HTTPS connections outside the cluster.",
 	)
 
+	flags.StringVar(
+		&args.noProxy,
+		"no-proxy",
+		"",
+		"A comma-separated list of domains and CIDRs for which the proxy should not be used.",
```

Comment:
> A comma-separated list of destination domain names, domains, IP addresses or other network CIDRs to exclude proxying.

(from the [docs](https://docs.openshift.com/container-platform/4.10/networking/enable-cluster-wide-proxy.html#nw-proxy-configure-object_config-cluster-wide-proxy))